test/model.js

```diff
@@ -989,8 +994,8 @@
 
   QUnit.test('`previous` for falsey keys', function(assert) {
     assert.expect(2);
-    var model = new Backbone.Model({0: true, '': true});
-    model.set({0: false, '': false}, {silent: true});
+    var model = new Backbone.Model({'0': true, '': true});
```

Comment:
It was a liter warning for inconsistent string usage. Must have committed it by accident. :relaxed: